src/test/regress/expected/multi_extension.out

```diff
@@ -112,6 +112,7 @@ ALTER EXTENSION citus UPDATE TO '6.2-3';
 ALTER EXTENSION citus UPDATE TO '6.2-4';
 ALTER EXTENSION citus UPDATE TO '7.0-1';
 ALTER EXTENSION citus UPDATE TO '7.0-2';
+ALTER EXTENSION citus UPDATE TO '7.0-3';
```

Comment:
I think we need a test testing that the moving of placements from one table to the other works.